docs/clojure/anonymous-function.md

```diff
@@ -96,10 +96,11 @@ If we use def to save the value...
 
 `combined` is exposed outside of our function, and this is bug-prone.
 
-~~~klipse
```

Comment:
@yokolet look at what I did here. This is due to a "bug" in self-host clojurescript: on some cases, we have to specify that the evaluation context is "expr" instead of "statement".
Another option is to wrap the expression into a vector.